charts/internal/cert-management-garden-runtime/templates/service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: cert-controller-manager
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    networking.resources.gardener.cloud/from-all-scrape-targets-allowed-ports: '[{"port":{{ .Values.configuration.serverPortHttp }},"protocol":"TCP"}]'
+    # TODO: This label approach is deprecated and no longer needed in the future. Remove them as soon as gardener/gardener@v1.75 has been released.
+    networking.resources.gardener.cloud/from-policy-pod-label-selector: all-scrape-targets
+    networking.resources.gardener.cloud/from-policy-allowed-ports: '[{"port":{{ .Values.configuration.serverPortHttp }},"protocol":"TCP"}]'
```

Comment:
🧹 Can we act on this `TODO` since `gardener/gardener` has just hit `v1.110`? :)

Reply:
Hm, let's try it. Following https://github.com/gardener/gardener-extension-registry-cache/pull/67 the only actions seems to be to remove the annotations.